pkg/internal/export/otel/metrics.go

```diff
@@ -46,6 +50,7 @@ const (
 
 	FeatureNetwork     = "network"
 	FeatureApplication = "application"
+	FeatureSpan        = "span"
```

Comment:
Being picky here. Isn't "span" a subset of "application" feature?

Reply:
I think you are right :), what should I call it? Maybe `trace_span_metrics`? Technically the `application` now is `application_otel` the new one is `application_trace_span_metrics`.

Reply:
Looking at it twice, I think it's fine just `span` or `application_span`.

Reply:
I like `application_span` a lot, thanks for the suggestion. It will remove confusion from the network metrics.